ontology/yaml/resources/HVAC/entity_types/PMP.yaml

```diff
@@ -60,6 +60,15 @@ PMP_SS_VSC_WDPM:
   - VSC
   - WDPM
 
+PMP_CPVSC2X_SWTM:
```

Comment:
The pump general type is intended to representative of a single pump rather than a set of pumps. Please model these pumps seperately and use a passthrough to contribute any shared points to both entities.

Reply:
@shambergoldstein - will do thanks a million for your assistance.